packages/test-app/src/App.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env browser */
```

Comment:
We probably don't want to add (and maintain) `test-app` in the repo? I usually just `yarn link` things when testing with real apps.

Reply:
Agreed 👍  I will remove this. I had left this in as a uniform test case for people to try whilst WIP 👍 

I think it could be worthwhile adding a `CONTRIBUTING.md` file that gives contributors a good idea of workflow etc.